openedx/features/custom_fields/multiselect_with_other/db/fields.py

```diff
@@ -104,6 +107,12 @@ def to_python(self, value):
         return MSFList(choices, [])
 
     def _check_other_max_length_attribute(self, **kwargs):
+        """
+        This function is to validate the MultiSelectWithOtherField has a
+        'other_max_length' attribute.
+        :param **kwargs: arguments
+        :type **kwargs: dictionary
```

Comment:
Please correct the information in docstring. 
Method is returning list of errors or empty list but docstring states that it is returning dictionary .